### PR TITLE
Add support for KHR_materials_unlit

### PR DESCRIPTION
**Pull request by adrien-ben**
Hi there,

I'd like to implement support for `KHR_materials_unlit` in my gltf viewer. Here is a PR adding support for it in gltf.

### CODE REVIEW

File: src/material.rs [hunk 1]
```diff
@@ -157,6 +157,17 @@ impl<'a> Material<'a> {
         self.json.emissive_factor.0
     }
 
+    /// Specifies whether the material is unlit.
+    /// 
+    /// * When this value is false, primitive should be renderer using Physically-Based Rendering (PBR) methodology.
+    /// * When this value is true, primitive should not be renderer using Physically-Based Rendering (PBR) methodology.
+    #[cfg(feature = "KHR_materials_unlit")]
+    pub fn is_unlit(&self) -> bool {
```
Comment (alteous) [inline src/material.rs]: Could we call this `unlit` instead?

```rust
pub fn unlit(&self) -> bool { ... }
```

File: src/material.rs [hunk 2]
```diff
@@ -157,6 +157,17 @@ impl<'a> Material<'a> {
         self.json.emissive_factor.0
     }
 
+    /// Specifies whether the material is unlit.
+    /// 
+    /// * When this value is false, primitive should be renderer using Physically-Based Rendering (PBR) methodology.
+    /// * When this value is true, primitive should not be renderer using Physically-Based Rendering (PBR) methodology.
```
Comment (alteous) [inline src/material.rs]: The `KHR_materials_unlit` is a suggestion to not use the 'core' PBR values. https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_unlit#overview

I suggest we reword something like:

```
Returns `true` if the [`KHR_materials_unlit`] property was specified, in which case the renderer should prefer to ignore all PBR values except `baseColor`.

[`KHR_materials_unlit`](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_unlit#overview)
```

File: src/material.rs (hunk 1 above)
Comment (adrien-ben) [reply src/material.rs]: Absolutly!

File: src/material.rs (hunk 2 above)
Comment (adrien-ben) [reply src/material.rs]: You're right, it's just a suggestion, I changed it. Thanks.

**Comment by adrien-ben** [comment]
Thanks :)

**Pull request merged**